OrbitCore/ElfFile.cpp

```diff
@@ -81,14 +92,70 @@ bool ElfFile::GetFunctions(Pdb* pdb, std::vector<Function>* functions) const {
     }
 
     // Limit list of symbols to functions. Ignore sections and variables.
-    if (symbol_ref->getType().get() != llvm::object::SymbolRef::ST_Function) {
+    if (symbol_ref.getType().get() != llvm::object::SymbolRef::ST_Function) {
       continue;
     }
 
     functions->emplace_back(name, Path::GetFileName(file_path_),
-                            symbol_ref->getValue(), symbol_ref->getSize(), pdb);
+                            symbol_ref.getValue(), symbol_ref.getSize(), pdb);
     function_added = true;
   }
 
   return function_added;
 }
+
+template <typename ElfT>
```

Comment:
It would be great to have a few lines explaining what is that "load bias", how we compute it and how we should use it.

Reply:
added a comment to the header file